scripts/validate_wheel.sh

```diff
@@ -141,6 +162,16 @@ for ex in `find "$root_folder/examples" -name '*.py' -not -path '*/providers/*'`
 done
 
 # Run remote-mqpu platform test
+# First determine the location of the cuquantum libraries (the [^, ] is to catch a suffix like "-cu11")
+CUQUANTUM_PACKAGE=$(python3 -m pip show --files cuda-quantum | grep Requires: | grep -o -P "cuquantum[^, ]*")
+CUQUANTUM_DEPS=$(python3 -m pip show --files $CUQUANTUM_PACKAGE | grep Requires: | cut -d' ' -f2- | tr -d ,)
+CUQUANTUM_LIB_PATH=$(for PACKAGE in $CUQUANTUM_DEPS; do
+  package_loc=$(python3 -m pip show --files $PACKAGE | grep Location: | cut -d' ' -f2)
+  package_so_dir=$(python3 -m pip show --files $PACKAGE | grep "\.so" | sed -re "s/ +//" | xargs dirname | head -1)
+  echo $package_loc/$package_so_dir
+done | sort | uniq | paste -d':')
+export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${CUQUANTUM_LIB_PATH}
+# Now continue with cudaq-qpud location
```

Comment:
@bettinaheim - I think this (i.e., updating validate_wheel.sh per these new lines) is the only way we can keep the `cuquantum` conda package out of the Python installation instructions. This means that pip package users would have to do something similar if they ever wanted to manually launch `cudaq-qpud` like we're doing in this test. Please let me know if you had something else in mind.

Reply:
I think we can clean this up a bit by including a launch script instead that automatically does the same thing as the __init__ file for the cudaq python package. We can make that script a python file, since I believe it would only be used by python users (other users don't need that script since there all necessary paths need to be set anyway).

Reply:
Just to confirm ... this Python-script approach won't work in the C++-only environment. Is that ok?

Reply:
Updated in 1a59a7251 and 2cd4f29fe.

Reply:
> Updated in [1a59a72](https://github.com/NVIDIA/cuda-quantum/commit/1a59a7251193517ab4627e6efc5315558390114c) and [2cd4f29](https://github.com/NVIDIA/cuda-quantum/commit/2cd4f29fe7e78c66889030f20a69c1d4faeab5ae).

Well, it looks like those didn't work. (Failed publishing run is here: https://github.com/NVIDIA/cuda-quantum/actions/runs/8697234380/job/23852549810). So I'll keep investigating.